src/app/scripts/cac/search/cac-geocoder.js

```diff
@@ -54,4 +56,27 @@ CAC.Search.Geocoder = (function ($) {
         return dfd.promise();
     }
 
+    function reverse(lat, lng) {
+        var dfd = $.Deferred();
+
+        var params = {
+            location: [lng, lat].join(','),
```

Comment:
May want to increase the default search radius (via the 'distance' parameter). I was playing around with dragging the markers and there were many times where I'd drop it in the middle of a park and it wouldn't return any results.